gef-ui/server/utils/mocks/mock_applications.js

```diff
@@ -765,6 +765,121 @@ const RES_MOCK_AIN_APPLICATION_CHECKER = () => {
+  comments: [],
+  previousStatus: CONSTANTS.DEAL_STATUS.UKEF_IN_PROGRESS,
+  ukefDecision: [],
+  ukefDecisionAccepted: true,
+  id: '61a7710b2ae62b0013dae687',
+  exporter: { status: CONSTANTS.DEAL_STATUS.COMPLETED, details: [], validation: [] },
+  facilities: {
+    status: CONSTANTS.DEAL_STATUS.COMPLETED,
+    items: [
+      {
+        status: CONSTANTS.DEAL_STATUS.COMPLETED,
+        details: {
+          _id: '61a771cc2ae62b0013dae68a',
+          dealId: '61a7710b2ae62b0013dae687',
+          type: 'CASH',
+          hasBeenIssued: false,
+          name: 'Facility two',
+          shouldCoverStartOnSubmission: true,
+          coverStartDate: 1638403200000,
+          coverEndDate: '2030-01-01T00:00:00.000Z',
+          monthsOfCover: null,
+          details: [],
+          detailsOther: '',
+          currency: 'GBP',
+          value: 2000,
+          coverPercentage: 80,
+          interestPercentage: 1,
+          paymentType: 'IN_ADVANCE_MONTHLY',
+          createdAt: 1638363596947,
+          updatedAt: 1638442632540,
+          ukefExposure: 1600,
+          guaranteeFee: 0.9,
+          submittedAsIssuedDate: '1638363717231',
+          ukefFacilityId: '0030113305',
+          feeType: 'in advance',
+          feeFrequency: 'Monthly',
+          dayCountBasis: 365,
+          coverDateConfirmed: false,
+        },
+        validation: { required: [] },
+      },
+      {
+        status: CONSTANTS.DEAL_STATUS.COMPLETED,
+        details: {
+          _id: '61a771cc2ae62b0013dae69a',
+          dealId: '61a7710b2ae62b0013dae687',
+          type: 'CASH',
+          hasBeenIssued: true,
+          name: 'Facility one',
+          shouldCoverStartOnSubmission: true,
+          coverStartDate: 1638403200000,
+          coverEndDate: '2030-01-01T00:00:00.000Z',
+          monthsOfCover: null,
+          details: [],
+          detailsOther: '',
+          currency: 'GBP',
+          value: 2000,
+          coverPercentage: 80,
+          interestPercentage: 1,
+          paymentType: 'IN_ADVANCE_MONTHLY',
+          createdAt: 1638363596947,
+          updatedAt: 1638442632540,
+          ukefExposure: 1600,
+          guaranteeFee: 0.9,
+          submittedAsIssuedDate: '1638363717231',
+          ukefFacilityId: '0030113305',
+          feeType: 'in advance',
+          feeFrequency: 'Monthly',
+          dayCountBasis: 365,
+          coverDateConfirmed: true,
+          canResubmitIssuedFacilities: true,
+        },
+        validation: { required: [] },
+      },
+    ],
+  },
+  exporterStatus: { text: 'Completed', class: 'govuk-tag--green', code: 'COMPLETED' },
+  eligibilityCriteriaStatus: { text: 'Completed', class: 'govuk-tag--green', code: 'COMPLETED' },
+  facilitiesStatus: { text: 'Completed', class: 'govuk-tag--green', code: 'COMPLETED' },
+  supportingInfoStatus: { text: 'Completed', class: 'govuk-tag--green', code: 'COMPLETED' },
+  canSubmit: false,
+  checkerCanSubmit: false,
+});
+
```

Comment:
I like that we have a few examples for various scenarios, but I'm wondering whether some of the properties can get inherited from each other? i.e. the properties that are common in all of them

Reply:
done